src/rpc/blockchain.cpp

```diff
@@ -994,7 +995,7 @@ UniValue getscinfo(const UniValue& params, bool fHelp)
         scId.SetHex(inputString);
  
         UniValue sc(UniValue::VOBJ);
-        if (!ScMgr::fillJSON(scId, sc) )
+        if (!AddScInfoToJSON(scId, sc) )
```

Comment:
in line 977 runtime error with help message returned, but no return data structure specified in it.

Reply:
Ok, will add it